src/main/java/com/manufacturing/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,120 @@
+package com.manufacturing.dao.jdbc;
+
+import com.manufacturing.dao.ManufacturerDao;
+import com.manufacturing.lib.Dao;
+import com.manufacturing.model.Manufacturer;
+import com.manufacturing.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    private static final String MANUFACTURER_NAME = "manufacturer_name";
+    private static final String MANUFACTURER_ID = "manufacturer_id";
+    private static final String ORIGIN = "origin";
```

Comment:
I think these constants is redundant because we use them only in the private method. But at the same time, we use `manufacturer_name `, `manufacturer_id `,  `origin ` in a lot of queries
So if we will change the name of the column, we have to change it in many places.